Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -927,6 +990,198 @@ def parse_port_range(port: str) -> tuple[Optional[int], Optional[int]]:
                 raise DemistoException(f"Security group {group_id} not found.")
             raise DemistoException(f"Failed to revoke egress rule: {e}")
 
+    @staticmethod
+    def create_security_group_command(client: BotoClient, args: Dict[str, Any]) -> CommandResults:
+        """
+        Creates a new security group in the specified VPC or EC2-Classic.
+
+        Args:
+            client (BotoClient): The boto3 client for EC2 service
+            args (Dict[str, Any]): Command arguments including:
+                - group_name (str): Name of the security group.
+                - description (str): Description of the security group.
+                - vpc_id (str, optional): VPC ID where security group will be created.
+
+        Returns:
+            CommandResults: Results of the operation with security group creation details
+        """
+        group_name = args.get("group_name")
+        description = args.get("description")
+        vpc_id = args.get("vpc_id")
+        kwargs = {"Description": description, "GroupName": group_name, "VpcId": vpc_id}
+        try:
+            remove_nulls_from_dictionary(kwargs)
+            response = client.create_security_group(**kwargs)
+            if response.get("ResponseMetadata", {}).get("HTTPStatusCode") == HTTPStatus.OK and (
+                group_id := response.get("GroupId")
+            ):
+                return CommandResults(
+                    readable_output=f'The security group "{group_id}" was created successfully.',
+                    raw_response=response,
+                )
+            else:
+                AWSErrorHandler.handle_response_error(response)
+
+        except ClientError as err:
```

Comment:
Why do we try:except inside the command and not in the main? In general it is best practice to use the try except in the outer layer.